sw/device/lib/crypto/impl/rsa/rsa_signature.c

```diff
@@ -222,7 +223,13 @@ status_t rsa_signature_verify_2048_finalize(
     hardened_bool_t *verification_result) {
   // Wait for OTBN to complete and get the size for the last RSA operation.
```

Comment:
nit: this comment should probably change in all 3 places in this file to indicate that it no longer waits for OTBN, but rather fails if OTBN isn't done.